### DISCUSSION TITLE
Thoughts on scrapers for Emby?

### DISCUSSION THREAD

**Discussion by j-ktz** [CodyBerenson/PGMA-Modernized#179] [Ideas]
I have been having  more  and more issues with  Plex and some of the features make it feel bloated. I've seen some folks jump to Emby. Would we  ever consider building scrapers  for it? Emby.media

**Comment by JPH71** [comment]
I think emby uses Java...
The best we will do would be to create nfo files that I am sure kodi and
emby recognise..

With the wirk I have being doing on GEVI..we are closer to realising this
xxx

**Comment by mhunter8** [comment]
Interesting... I've been playing with Emby, but it has it's own weird issues, for example I was traveling and couldn't get a reliable streaming connection to my home server, but when I switched back to Plex it worked with no issues.  They also have some weird quirks, for example if you have an ampersand =>  &  =<  in your nfo file, it MUST be changed to => `&amp;` <= or it is displayed incorrectly.  I had to write a quick script to go though my entire plex library to make that change.  It wasn't hard, just annoying.  

One the plus side however, and IMO this is a big plus, emby is much more straight forward with the way they handle and store their metadata, it doesn't go into some cryptic by design Plex database menagerie.  Also, the Cast avatars are larger with emby, so you can get face and torso rather than just a head shot.  Also right now plex has this bizarre issue with the android tv client where it won't display artwork at all for local cast.    However, the cast is displayed perfectly fine on their webclient and Roku client.
